shared/types/bridge-status.ts

```diff
@@ -167,6 +160,10 @@ export type StartPollingForBridgeTxStatusArgsSerialized = Omit<
 
 export type SourceChainTxMetaId = string;
 
-export type BridgeStatusControllerState = {
+export type BridgeStatusState = {
   txHistory: Record<SourceChainTxMetaId, BridgeHistoryItem>;
 };
+
+export type BridgeStatusControllerState = {
+  bridgeStatusState: BridgeStatusState;
+};
```

Comment:
- Rename `BridgeStatusControllerState` to `BridgeStatusState`
- Define new `BridgeStatusControllerState`